pkg/minishift/config/viper_config.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright (C) 2018 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/minishift/minishift/pkg/minikube/constants"
+	"io"
+	"os"
+)
+
+type ViperConfig map[string]interface{}
```

Comment:
Not following the name. Why do we call this `ViperConfig` ?

Reply:
@gbraad because now this config only for viper related stuff, not anything else so I named it to `ViperConfig`.